golang/src/user-sessions/main.go

```diff
@@ -0,0 +1,393 @@
+package main
+
+import (
+	"configurate"
+	"database/sql"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"logcabin"
+	"net/http"
+	"os"
+	"queries"
+	"strings"
+
+	"github.com/gorilla/mux"
+	_ "github.com/lib/pq"
+)
+
+var (
+	version = flag.Bool("version", false, "Print the version information")
+	cfgPath = flag.String("config", "/etc/iplant/de/jobservices.yml", "The path to the config file")
```

Comment:
This is not a job-related service except in the very vague sense that all our services sort of revolve around jobs at *some* level, so this seems like an odd default. We can always pass this name via our deployment since I believe you're doing it this way because that file will already have the appropriate DB configuration, but for the default I'd prefer for it to be a name that's more obvious, for downstream folks who might want to use this.